test/automated_generation_e2e_test.ts

```
@@ -0,0 +1,745 @@
+        }
+        if (attempts === 1) {
+          console.log(`Waiting for Electron app to start (max ${CONFIG.CDP_MAX_ATTEMPTS} attempts)...`);
+        }
+        await new Promise((resolve) => setTimeout(resolve, CONFIG.CDP_RETRY_DELAY));
+      }
+    }
+
+    // Get page
+    const contexts = resources.browser!.contexts();
+    console.log(`Found ${contexts.length} browser contexts`);
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)